lib/triggers/queryOrders.js

```diff
@@ -55,14 +72,50 @@ function processTrigger(msg, cfg, next, snapshot) {
             .fetch();
     }
 
+    function addReboundedOrders(ordersResponse) {
```

Comment:
As we don't really perform a ``rebound`` here, let's use a better wording. I think it make sense to replace the occurrences of the word ``rebound`` by ``orderOfNonSyncedCustomer``